scripts/west_commands/sign.py

```diff
@@ -391,6 +391,20 @@ def edt_flash_params(flash):
 
 class RimageSigner(Signer):
 
+    @staticmethod
+    def edt_get_rimage_target(board):
+        if 'up_squared_adsp' in board:
```

Comment:
shouldn't this be intel_adsp_cavs15 here? There are many other APL/GPL boards that are not up2.

Reply:
yes, was wondering about changing it and started doing that, but ended up deciding to get this patch series as is now and do the migration to the same naming convention later. This board exists already in zephyr with this name.

Reply:
sounds fine @nashif